alerts/kubelet.libsonnet

```diff
@@ -48,6 +48,20 @@
               message: "Kubelet '{{ $labels.node }}' is running at {{ $value | humanizePercentage }} of its Pod capacity.",
             },
           },
+          {
+            alert: 'KubeletPodStartUpLatencyIsHigh',
+            expr: |||
+              histogram_quantile(0.99, sum(rate(kubelet_pod_worker_duration_seconds_bucket{%(kubeletSelector)s}[5m])) by (instance, le)) > 5
```

Comment:
Could we join the kubelet_node_name metric o to this so we can use the node name in the message? Then we can also aggregate by node name instead of instance label.